draft-ietf-scitt-software-use-cases.md

```diff
@@ -203,14 +203,14 @@ There is no standardized way to:
 
 ## Promotion of a Software Component by Multiple Entities
 
-A software component source (e.g., a library) released by a certain original producer is becoming popular.
-The released software component source is accompanied by a statement of authenticity (e.g., a detached signature).
+A software component (e.g., a library) released by a certain original producer is becoming popular.
+The released software component is accompanied by a statement of authenticity (e.g., a detached signature).
 Over time, due to its enhanced applicability to various products, there has been an increasing amount of multiple providers of the same software component version on the internet.
 
 Some providers include this particular software component as part of their release package bundle and provide the package with proof of authenticity using their own issuer authority.
 Some packages include the original statement of authenticity, and some do not.
-Over time, some providers no longer offer the exact same software component source but pre-compiled software component binaries.
-Some sources do not provide the exact same software component but include patches and fixes produced by third-parties, as these emerge faster than solutions from the original producer.
+Over time, some providers no longer offer the exact same software component source code but pre-compiled software component binaries.
```

Comment:
Minor, non-blocking: so in this case, I would mention *source code* because that is where it is meant and casually referring to it as source (as a place or origin or maybe shorthand for source code) was confusing me here.